### PR TITLE
Block add capabilities

### LINKED ISSUE #398: MTB: Add test for Block add capabilities
Automated test for https://github.com/kubernetes-sigs/multi-tenancy/blob/master/benchmarks/e2e/tests/block_add_capabilities/README.md

### PR DESCRIPTION
**Pull request by SomtochiAma**
Closes #398

### CODE REVIEW

File: benchmarks/go.mod [hunk 1]
```diff
@@ -40,6 +40,7 @@ require (
 	github.com/prometheus/common v0.7.0
 	github.com/prometheus/procfs v0.0.5
 	github.com/realshuting/multi-tenancy-benchmarks v0.0.0-20191028041724-a27250830445
+	github.com/realshuting/multi-tenancy/benchmarks v0.0.0-20200221171633-2b5c81967af0
```
Comment (rjbez17) [inline benchmarks/go.mod:43]: Shouldn't these point to this repo and not @realshuting's fork?

File: benchmarks/go.sum [hunk 2]
```diff
@@ -359,7 +359,10 @@ github.com/prometheus/procfs v0.0.2/go.mod h1:TjEm7ze935MbeOT/UhFTIMYKhuLP4wbCsT
 github.com/prometheus/procfs v0.0.5 h1:3+auTFlqw+ZaQYJARz6ArODtkaIwtvBTx3N2NehQlL8=
 github.com/prometheus/procfs v0.0.5/go.mod h1:4A/X28fw3Fc593LaREMrKMqOKvUAntwMDaekg4FpcdQ=
 github.com/quobyte/api v0.1.2/go.mod h1:jL7lIHrmqQ7yh05OJ+eEEdHr0u/kmT1Ff9iHd+4H6VI=
+github.com/realshuting/multi-tenancy v0.0.0-20200221171633-2b5c81967af0 h1:xoaP0/13R5JO/FhDP8PAdL1OKh5a03CKDvKGOaVU6Rc=
```
Comment (rjbez17) [inline benchmarks/go.sum:362]: Shouldn't these point to this repo and not @realshuting's fork?

File: benchmarks/go.mod (hunk 1 above)
Comment (SomtochiAma) [reply benchmarks/go.mod:43]: I don't think any changes were supposed to be made to the go.mod and go.sum files. I will revert the,

**Comment by rjbez17** [comment]
@SomtochiAma will you have an opportunity to rebase this?

**Comment by SomtochiAma** [comment]
> @SomtochiAma will you have an opportunity to rebase this?

Sure. Doing it now

**Pull request merged**